Dockerfile.dev

```
@@ -1,4 +1,4 @@
-FROM python:3.11-alpine
+FROM python:3.11.9-alpine3.20
```

Let's use a floating tag to benefit from Python security patches.

```suggestion
FROM python:3.11-alpine3.20
```

Alternatively, if we do fix the Python version, let's set up a bot that updates the tag.

---

Done